### DISCUSSION TITLE
Add e AddRange com mensagem de erro no curso Fundamentos da Orientação a Objeto

### DISCUSSION THREAD

**Discussion by amarinafroes** [balta-io/forum#317] [Tira Dúvidas] [answered]
Na aula do video Notificações e Herança, onde o Professor usa o Add e AddRange, para mim aparece uma mensagem de erro:

![image](https://user-images.githubusercontent.com/89412365/234351159-eff0b4a2-a95c-4812-9d0b-31fb22eace52.png)

![image](https://user-images.githubusercontent.com/89412365/234351244-7ef747a1-e891-4c2b-9d0c-1f7238086c4d.png)

mesmo usando List ao invés do IList segue meu código :

using System.Collections.Generic;

namespace Marina.NotificationContext
{
    public abstract class Notifiable
    {
        public List<Notification> Notifications { get; set; }
        public void Add(Notification notification)
        {
            Notification.Add(notification);
        }
        public void AddRange(IEnumerable<Notification> notifications)
        {
            Notification.AddRange(notifications);
        }
    }
}

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @amarinafroes, como vai?

Acredito que seja um "problema" relacionado a versão do Flunt! 
Você consegue me dizer qual versão está utilizando?

Acredito que a versão do curso seja a 1.0.5!
